### DISCUSSION TITLE
Add Difficulty Owner Information In-Game

### DISCUSSION THREAD

**Discussion by krsmll** [ppy/osu#26984] [Ideas]
Currently, the beatmapset owner is displayed as the mapper. However, it would be neat to display guest difficulty mapper information separately as well, potentially even with a navigatable link that opens up user's profile.

![no difficulty owner in game](https://github.com/ppy/osu/assets/57319032/5a75d2e1-9fab-48ee-afe1-21f62194c7fd)
![no difficulty owner](https://github.com/ppy/osu/assets/57319032/f5852554-98b6-4dd4-9c1f-24c4f4aa740d)

You already have that information in web so I don't think it is very difficult to implement.

I am aware that work is done regarding the song selection panel and saw the drafts. However, I wasn't sure if this is something already addressed as there is a lot of filler text which is understandable.

**Comment by Flamiii** [comment]
If you were to click on the "mapped by riot1133", it would actually send you to the guest difficulty owner's profile (Lotragon). The name displayed not being correct is a known bug that is being tracked (https://github.com/ppy/osu/issues/12965).

**Reply by krsmll** [reply]
I am aware of it. There is already a bug report of this so I did not mention this here.

**Reply by Flamiii** [reply]
If what you're asking for is to display both the host and the guest mapper in song select, that probably won't happen as peppy doesn't deem it necessary (https://github.com/ppy/osu/discussions/19962#discussioncomment-3480551). I don't really think it's necessary either to be honest.

**Reply by krsmll** [reply]
I think the bug report had a good idea.

`hosted by <host> and mapped by <creator>`
